catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/MatchAnyValidator.java

```diff
@@ -37,42 +39,59 @@ public MatchAnyValidator(List<AttributeValidator> validators) {
       return Optional.empty();
     }
 
-    List<Optional<AttributeValidationReport>> validationReportList =
-        validators
-            .stream()
-            .map(validator -> validator.validate(attribute))
-            .collect(Collectors.toList());
+    List<Serializable> attributeValues = attribute.getValues();
+    List<AttributeValidationReport> resultValidationReportList = new ArrayList<>();
 
-    return generateValidationReports(validationReportList);
+    for (Serializable serializable : attributeValues) {
+
+      List<AttributeValidationReport> validationReportList =
+          validateAttributeSerializable(attribute.getName(), serializable);
+
+      if (validationReportList.size() == validators.size()) {
+        resultValidationReportList.addAll(validationReportList);
+      }
+    }
+
+    return generateValidationReports(resultValidationReportList);
+  }
+
+  private List<AttributeValidationReport> validateAttributeSerializable(
+      String attributeName, Serializable serializable) {
+    List<AttributeValidationReport> validationReportList = new ArrayList<>();
+    for (AttributeValidator attributeValidator : validators) {
+      Attribute newAttribute = new AttributeImpl(attributeName, serializable);
```

Comment:
The `Attribute` could be created once before the loop and reused.

Reply:
fixed